v2/tools/generator/internal/astmodel/external_package_reference.go

```diff
@@ -64,11 +59,6 @@ func (pr ExternalPackageReference) String() string {
 	return pr.packagePath
 }
 
-// TryGroupVersion returns the group and version of this external reference.
-func (pr ExternalPackageReference) TryGroupVersion() (string, string, bool) {
-	return "", "", false
-}
-
 // GroupVersion triggers a panic because external references don't have a group or version
 func (pr ExternalPackageReference) GroupVersion() (string, string) {
```

Comment:
Does it make sense to remove this too?

Reply:
If memory serves, there were a couple of outstanding references - and I thought this PR had gotten large enough. Removing this is a goal, but not in this PR.